searchcore/src/tests/proton/attribute/attribute_test.cpp

```diff
@@ -975,6 +980,15 @@ TEST_F(AttributeWriterTest, forceCommit_clears_search_cache_in_imported_attribut
     EXPECT_EQ(0u, _mgr->getImportedAttributes()->get("imported_b")->getSearchCache()->size());
 }
 
+TEST_F(AttributeWriterTest, ignores_force_commit_serial_not_greater_than_create_serial)
+{
+    auto a1 = addAttribute("a1");
+    a1->setCreateSerialNum(100);
+    EXPECT_EQ(0u, test_force_commit(*a1, 50u));
+    EXPECT_EQ(0u, test_force_commit(*a1, 100u));
+    EXPECT_EQ(150u, test_force_commit(*a1, 150u));
```

Comment:
In this case I would test with 101, as that is the first value with changed behaviour.